### ISSUE TITLE
Printing with windows line ending

### ISSUE THREAD

**Issue by germtb** [benjamn/recast#262]
`recast.print(ast).code;` seems to use the line ending of the operative system I am working on (windows), but I depend on 
```
equal(print(convertedAST).code, expectedCode)
```
Where `expectedCode` is loaded from a file with unix line endings. I could change those files but then the project would not work on unix. Is it possible to configure `print` to handle use one line ending or the other?

**Comment by benjamn** [comment]
`recast.print` tries to change as little as possible about the source code, and changing all the line endings would be a pretty big change. Though I see the value of what you're trying to do, I think you're probably better off just post-processing the code with something like `recast.print(ast).code.replace(/\r\n/g, "\n")`.

**Comment by germtb** [comment]
Yep, that's what I ended up doing. I guess it makes sense for print to do as little as possible, and doing a replace is quite easy.
